ethcore/builtin/src/lib.rs

```diff
@@ -105,26 +113,32 @@ struct AltBn128PairingPrice {
 }
 
 /// alt_bn128_pairing pricing model. This computes a price using a base cost and a cost per pair.
+#[derive(Debug)]
 struct AltBn128PairingPricer {
 	price: AltBn128PairingPrice,
-	eip1108_transition_at: u64,
-	eip1108_transition_price: AltBn128PairingPrice,
 }
 
-impl Pricer for AltBn128PairingPricer {
-	fn cost(&self, input: &[u8], at: u64) -> U256 {
-		let price = if at >= self.eip1108_transition_at {
-			self.eip1108_transition_price
-		} else {
-			self.price
-		};
+/// Pricing for constant alt_bn128 operations
+#[derive(Debug, Copy, Clone)]
+pub struct AltBn128ConstOperations {
+	/// price
+	pub price: usize,
```

Comment:
It is a bug, should be u64.

Reply:
No, not really, I think I used it because the other pricing models used `usize`.

Are you thinking of `platform-dependent` issues when `price > u32::max_value()`?

Reply:
yeah, just wondering why not `u64` in the first place

Reply:
that's an excellent observation, shall I address it in this PR or create an issue?